modules/restore-remove-failed-master.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-remove-failed-master_{context}"]
+= Removing a failed master host from the etcd cluster
+
+Follow these steps to remove a failed master host from the etcd cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* SSH access to an active master host.
+
+.Procedure
+
+. Access an active master host.
+
+. View the list of Pods associated with etcd:
++
+----
+[core@ip-10-0-128-73 ~]$ oc get pods -n openshift-etcd
```

Comment:
Same question as above, do we need the command prompt before `$` in these steps?